docker/preview/Jenkinsfile

```diff
@@ -293,12 +308,20 @@ node("docker-daemon") {
         parallel (
           "app": {
             dir("app") {
-              createTag(nextAppReleaseTag)
+              if (params.source != 'latest') {
+                createTag(nextAppReleaseTag)
+              } else {
+                println("Source: latest; skip creating a tag")
+              }
             }
           },
           "config": {
             dir("config") {
-              createTag(nextConfigReleaseTag)
+              if (params.source != 'latest') {
```

Comment:
Would not it be better to reverse this and have `== latest` in `if` and then everything `else` ? It sounds more human `if it's latest, then skip creating a tag, otherwise, create a tag` :)